packages/nimble-components/src/rich-text-viewer/styles.ts

```diff
@@ -1,12 +1,60 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
-import { bodyFont, bodyFontColor } from '../theme-provider/design-tokens';
+import {
+    bodyFont,
+    bodyFontColor,
+    linkActiveFontColor,
+    linkFontColor,
+    smallPadding
+} from '../theme-provider/design-tokens';
 
 export const styles = css`
     ${display('flex')}
 
     :host {
         font: ${bodyFont};
+        outline: none;
         color: ${bodyFontColor};
+        inline-size: auto;
+        overflow: auto;
+        block-size: 100%;
+        min-block-size: 36px;
+    }
+
+    .viewer {
+        font: inherit;
+        outline: none;
+        box-sizing: border-box;
+        position: relative;
+        color: inherit;
+        min-inline-size: 100px;
+        padding: ${smallPadding};
+        margin-block-end: 10px;
+        margin-inline-end: 10px;
+    }
+
+    .viewer > :first-child {
+        margin-block-start: 0;
+    }
+
+    .viewer > :last-child {
+        margin-block-end: 0;
+    }
+
+    li > p {
+        margin-block: 0;
+    }
+
+    li > p:empty {
+        display: contents;
```

Comment:
If the p tag is empty it doesn't have any contents so not clear what sizing to the contents is supposed to do... What's the goal here? Just hiding the element? Should this be display:none?

Reply:
Yes, the goal is to hide the element and it should be `display:none` as I am checking if it is `empty` and updating the style. The actual reason to do this styling is that only in the Firefox browser, numbered lists overlap because `<p>` within `<li>` default margin is disabled. By using `display:contents`, the `<p>` element is omitted and its contents become part of the `<li>` layout.